language/unit_tests/test_syntax.py

```diff
@@ -125,3 +126,17 @@ def test_from_api_repr(self):
         sentence = klass.from_api_repr(payload)
         self.assertEqual(sentence.content, content)
         self.assertEqual(sentence.begin, begin)
+        self.assertEqual(sentence.sentiment, None)
+
+        score = 0.5
```

Comment:
Sorry I didn't mention before. Can you make this new part its own unit test case (import will go there to)?

Other than that this PR looks LGTM